examples/transposed3d.py

```diff
@@ -0,0 +1,60 @@
+import numpy as np
```

Comment:
I think these should actually be tests. Could you move them to dir `test` and make them proper pytest tests?